library/std/src/sys/os_str/wtf8.rs

```diff
@@ -1,5 +1,6 @@
 //! The underlying OsString/OsStr implementation on Windows is a
 //! wrapper around the "WTF-8" encoding; see the `wtf8` module for more.
+#![deny(unsafe_op_in_unsafe_fn)]
```

Comment:
Some places have been doing `forbid` if there aren't any `allow` exceptions, does that work here?

Reply:
I've switched it to forbid and moved it to the parent module, which seems to work for both Windows and Linux (which should cover the two branches in `librarry/std/src/sys/os_str/mod.rs`).